ee/codegen/src/generators/workflow.ts

```diff
@@ -34,8 +34,6 @@ import { DictEntry, isDefined } from "src/utils/typing";
 
 export declare namespace Workflow {
   interface Args {
-    /* The name of the module that the workflow class belongs to */
-    moduleName: string;
```

Comment:
big part of this PR was getting rid of as many `moduleName` references as possible. We should always consider modules as paths.